src/XMLGenerator.php

```diff
@@ -66,9 +66,13 @@ public function createUser(
         }
         if (!empty($user->getEmail())) {
             $info->addChild('Email', $user->getEmail());
+        } else {
+            $info->addChild('Email');
```

Comment:
I see that adding the empty `EmployeeID` node required a unit test change (as could be expected) however adding the empty `Email` node does not? Are we missing something in the unit tests?